### PR TITLE
GO-2050 Change metrics proposal

### PR DESCRIPTION
**Pull request by fat-fellow**
---

- [ ] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: metrics/client.go [hunk 1]
```diff
@@ -134,6 +138,8 @@ func (c *client) sendNextBatch(info appInfoProvider, b *mb.MB[amplitude.Event],
 			With("user-id", info.getUserId()).
 			With("messages sent", len(msgs)).
 			Debug("events sent to amplitude")
+		//flush arena
```
Comment (fat-fellow) [inline metrics/client.go:141]: @cheggaaa

File: metrics/amplitude/client.go [hunk 2]
```diff
@@ -3,64 +3,46 @@ package amplitude
...
+type Event *fastjson.Value
 
 // New client with API key
 func New(eventEndpoint string, key string) *Client {
+	arenaPool := &fastjson.ArenaPool{}
 	return &Client{
 		eventEndpoint: eventEndpoint,
 		key:           key,
 		client:        new(http.Client),
+		Arena:         arenaPool.Get(),
```
Comment (fat-fellow) [inline metrics/amplitude/client.go:30]: I do not put it back because it persists until the application is closed. @cheggaaa

Comment (deff7) [reply metrics/amplitude/client.go:30]: then what a purpose?

Comment (fat-fellow) [reply metrics/amplitude/client.go]: didn't understand how it works at first, simply created an arena

**Pull request merged**